finance.go

```diff
@@ -0,0 +1,185 @@
+package gofakeit
+
+import (
+	"math/rand"
+	"strconv"
+	"unicode"
+)
+
+// Reference Constants
+const cusipRunes = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
In the helpers.go file is a bunch of consts you can use. For this you should be able to just use the upperStr + numericStr, so that should help not needing an extra one.

Reply:
Thanks!  Will make this change.